src/Mod/Fem/femcommands/manager.py

```diff
@@ -380,7 +379,9 @@ def add_filter_set_edit(self, filtertype):
         # like add_obj_on_gui_selobj_noset_edit but the selection is kept
         # and the selobj is expanded in the tree to see the added obj
 
-        # check if we should use python filter
+        # check if we should use python fitler
```

Comment:
I don't understand this change, what's a fitler?

Reply:
neither do I :)  Most likely happened during refactoring somehow, will change it back.